### PR TITLE
[openvdb] update to 8.2.0

### LINKED ISSUE #21095: [OpenVDB] Update to V9.0 (from 7.x)
****

**Is your feature request related to a problem? Please describe.**
The openVDB port is now two full versions behind the current release.   NanoVDB has also been officially released as part of V9.0 to allow for GPU support.

**Proposed solution**
Update the vcpkg version would...well...make the library up-to-date.

**Describe alternatives you've considered**
Waiting so far hasn't gotten it updated so I am putting this request in.

**Additional context**
V9.0:
-------------------------------------------
NanoVDB, which offers GPU support
Improved compilation times
Support for OpenEXR version 3

### PR DESCRIPTION
**Pull request by Adela0814**
Fix [#21095](https://github.com/microsoft/vcpkg/issues/21095)
[openvdb] update to 8.2.0
Fix the case mismatch issue in cmakelist. This variable is defined in portfile of TBB port, which is TBBVERSION instead of TbbVERSION.

All features have been tested successfully in the following triplet:

x86-windwos
x64-windwos
x64-windows-static

### CODE REVIEW

File: ports/openvdb/fix-case-sensitive.patch [hunk 1]
```diff
@@ -0,0 +1,13 @@
+diff --git a/openvdb/openvdb/CMakeLists.txt b/openvdb/openvdb/CMakeLists.txt
+index dc33b72..08b6bb3 100644
+--- a/openvdb/openvdb/CMakeLists.txt
++++ b/openvdb/openvdb/CMakeLists.txt
+@@ -129,7 +129,7 @@ endif()
+ 
+ find_package(TBB CONFIG REQUIRED)
+ if(OPENVDB_FUTURE_DEPRECATION AND FUTURE_MINIMUM_TBB_VERSION)
+-  if(${Tbb_VERSION} VERSION_LESS FUTURE_MINIMUM_TBB_VERSION)
++  if(${TBB_VERSION} VERSION_LESS FUTURE_MINIMUM_TBB_VERSION)
```
Comment (JackBoosY) [inline ports/openvdb/fix-case-sensitive.patch:10]: Can you please merge this to patch `0003-fix-cmake.path`?

Thanks.

**Review by JackBoosY — CHANGES_REQUESTED** [review]

File: ports/openvdb/fix-case-sensitive.patch (hunk 1 above)
Comment (Adela0814) [reply ports/openvdb/fix-case-sensitive.patch]: Done

Comment (Adela0814) [reply ports/openvdb/fix-case-sensitive.patch]: Done

**Review by JackBoosY — APPROVED** [review]

**Comment by BillyONeal** [comment]
Thanks for the update!

**Pull request merged**